### DISCUSSION TITLE
HoverCard not working

### DISCUSSION THREAD

**Discussion by sandeepsre** [mantinedev/mantine#1955] [Q&A]
Trying to implement HoverCard into my current project. It's give an error
![image](https://user-images.githubusercontent.com/16818395/182206096-40d3b05d-2b34-4137-bd9a-b4a57f337c16.png)

**Comment by sandeepsre** [comment]
how to use HoverCard?

**Comment by rtivital** [comment]
To you HoverCard you need to update all your `@mantine/` packages to the latest version (5.0.2).
